### PR TITLE
removed unchecked casts

### PR DESCRIPTION
**Pull request by MathisGD**
before

image

after

image

### CODE REVIEW

File: src/Morpho.sol [hunk 1]
```diff
@@ -479,7 +476,7 @@ contract Morpho is IMorpho {
         }
 
         // Safe "unchecked" cast.
```
Comment (Rubilmax) [inline src/Morpho.sol:478]: This comment can be removed

File: src/Morpho.sol [hunk 2]
```diff
@@ -144,7 +143,7 @@ contract Morpho is IMorpho {
         require(market[id].lastUpdate == 0, ErrorsLib.MARKET_ALREADY_CREATED);
 
         // Safe "unchecked" cast.
```
Comment (Rubilmax) [inline src/Morpho.sol:145]: This comment can be removed

File: src/Morpho.sol [hunk 3]
```diff
@@ -378,15 +377,13 @@ contract Morpho is IMorpho {
         if (position[id][borrower].collateral == 0) {
             badDebtShares = position[id][borrower].borrowShares;
             // Safe "unchecked" cast since the value is smaller than totalBorrowAssets.
```
Comment (Rubilmax) [inline src/Morpho.sol:379]: This comment can be removed

**Review by Rubilmax — COMMENTED** [review]
I'm not strongly against as I understand we're moving to a "safer" code but strictly speaking it's just adding code that we know from reasoning will never be actually used (so we're adding some dead code)

So it goes against minimalism to me ^^

File: src/Morpho.sol (hunk 2 above)
Comment (MathisGD) [reply src/Morpho.sol:145]:
```suggestion
```

File: src/Morpho.sol (hunk 1 above)
Comment (MathisGD) [reply src/Morpho.sol:478]:
```suggestion
```

**Pull request closed without merging**